runtime/resolvers/resolvers_test.yaml

```diff
@@ -0,0 +1,102 @@
+project:
+  sources:
+    ad_bids_mini_source.yaml:
+      connector: local_file
+      path: AdBids_mini.csv
+  models:
+    "ad_bids_mini.sql": | 
+      select
+        id,
+        timestamp,
+        publisher,
+        domain,
+        volume,
+        impressions,
+        clicks
+      from ad_bids_mini_source
+  dashboards:
+    ad_bids_mini_metrics_with_policy.yaml:
+      model: ad_bids_mini
+      display_name: Ad bids
+      description:
+
+      timeseries: timestamp
+      smallest_time_grain: ""
+
+      dimensions:
+      - label: Publisher
+        name: publisher
+        expression: upper(publisher)
+        description: ""
+      - label: Domain
+        property: domain
+        description: ""
+
+      measures:
+      - label: "Number of bids"
+        name: bid's number
+        expression: count(*)
+      - label: "Total volume"
+        name: total volume
+        expression: sum(volume)
+      - label: "Total impressions"
+        name: total impressions
+        expression: sum(impressions)
+      - label: "Total clicks"
+        name: total click"s
+        expression: sum(clicks)
+
+      security:
+        access: true
+        row_filter: "domain = '{{ .user.domain }}'"
+        exclude:
+          - if: "'{{ .user.domain }}' != 'msn.com'"
+            names: 
+              - total volume
+  apis:
+    mv_sql_policy_api.yaml:
+      kind : api
+      metrics_sql: | 
+        select 
+            publisher,
+            domain, 
+            "total impressions", 
+            "total volume" 
+        FROM 
+          ad_bids_mini_metrics_with_policy 
+connectors:
+   duckdb: ~
+   clickhouse: ~
+tests:
+  simple:
+    resolver: mv_sql_policy_api 
+    options:
+      claims:
+        user_attributes:
+          domain: yahoo.com 
+          email: user@yahoo.com
+    result:
+      - domain: yahoo.com
+        publisher: YAHOO
+        "total volume": ~
+        "total impressions": 3.0
+  msn:
+    resolver: mv_sql_policy_api 
+    options:
+      claims:
+        user_attributes:
```

Comment:
For the sake of cleaner tests, I think we should flatten the nesting here and just have `user_attributes` as a root property of the test type

Reply:
It relates to the previous argument If you like to have more test structures then it's less production code coverage.

Reply:
I understand that we would like to see a more concise schema but it's already have a diverse structure, simplifying it is short term goal - eventually we will need to test more production fields anyway.

Reply:
This doesn't impact production code coverage at all – it just makes the test fixture simpler to read/write. Internally the test fixture parser would create a `SecurityClaims` value, so the production code coverage is exactly the same.

Reply:
Done.